docs/diagrams/FindClassDiagram.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Class "FindCommand" as FindCommand
+Command <|-- FindCommand
+FindCommand --righto "1" Predicate : contains
```

Comment:
Should the aggregation be on the other side instead? FindCommand contains a Predicate.